grails-app/domain/org/pih/warehouse/requisition/RequisitionItem.groovy

```diff
@@ -680,7 +690,7 @@ class RequisitionItem implements Comparable<RequisitionItem>, Serializable {
         }
     }
 
-    def getShipmentItems() {
+    Collection<ShipmentItem> getShipmentItems() {
```

Comment:
I know there's a temptation to make these changes (I do it as well) but let's try to avoid doing things like this just so we don't have to chase regressions for things that weren't a problem. At some point I'm willing to go back through and make wholesale changes but let's leave working code alone.